pkg/operator/operator.go

```diff
@@ -132,19 +118,3 @@ func (o *Operator) Run() error {
 		}
 	}
 }
-
-func (o *Operator) initResources() error {
-	kitCtx := opkit.Context{
-		Clientset:             o.context.Clientset,
-		APIExtensionClientset: o.context.APIExtensionClientset,
-		Interval:              500 * time.Millisecond,
-		Timeout:               60 * time.Second,
-	}
-
-	// Create and wait for CRD resources
-	err := opkit.CreateCustomResources(kitCtx, o.resources)
```

Comment:
what about TPRs? Does the opkit do both CRDs and TPRs. cc @travisn

Reply:
Yes, this will remove `TPR`'s. Bummer. Will fix.

Reply:
Added `TPR` `rook-operator` variant.